routes/user.py

```diff
@@ -179,11 +199,24 @@ def basic_register_new_user():
 
     roles = [session.query(Role).get(RoleName.user.value)]
 
+    avatar = Identicon()
```

Comment:
I think we need to move this duplication avatar generating code into some function